README.md

```diff
@@ -107,6 +107,23 @@ Then, open `chrome://inspect` in Chrome to attach the debugger (look into the "R
 
 This project is set up to use [jest](https://facebook.github.io/jest/) for tests. You can configure whatever testing strategy you like, but jest works out of the box. Create test files in directories called `__tests__` or with the `.test.js` extension to have the files loaded by jest. See an example test [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/app/App.test.js). The [jest documentation](https://facebook.github.io/jest/docs/en/getting-started.html) is also a wonderful resource, as is the [React Native testing tutorial](https://facebook.github.io/jest/docs/en/tutorial-react-native.html).
 
+## UI Tests
+
+This repository uses appium to run UI tests. The tests live in `__device-tests__` and are written using Appium to run tests against simulators and real devices. To run these you'll need to check off a few things: 
+
+* For now you'll need run `yarn start`, and then either `yarn ios` or `yarn android` at least once before trying to run the tests on the respective platform
```

Comment:
I wonder if we still need the `run-yarn-start-at-least-once` manual step. Do we? This PR includes scripts for CI and apparently things are done automatically there so, I wonder about the manual step in this case.

Reply:
Locally for now yes... it uses the debug build locally and the release build in CI... Debug build isn't bundled with jsbundle and as a result, requires those steps.

Reply:
I see, thanks.

No need to be part of this PR but, I wonder if trying to build a debug bundle and use that locally for device tests is a more robust strategy. Manual steps that only need to be done in cases that's not exactly clear when, tend to become gotchas and lead to a frustrating dev experience. Would be similar to the Jest tests anyway, since those don't use the running Metro instance either. Am I perhaps missing some other detail here?

Reply:
Hmm I'm not sure but I don't think that bundle would be rebuilt when changes are made(?), which I think is a plus when developing the tests locally to not have to rebundle after small changes and allow for the hot reloading to update the app and simply run the test again after the change, I mean we could have it as a pre-step but then that adds a little more time to running locally.. If I'm wrong about my first statement then sure this could work well

Reply:
> I think is a plus when developing the tests locally to not have to rebundle after small changes and allow for the hot reloading to update the app

Fair point and one I can totally relate to.

Ultimately, my main concern here is the manual step to install the native binary. How about always building and installing the binary (effectively, always running `yarn android` or `yarn ios`) so any changes to the native side can be picked up too? To be clear, this is not something to be tacked necessarily in this PR, it's not a blocker.